### ISSUE TITLE
Structure search is too difficult to find

### ISSUE THREAD

**Issue by aekhendry** [thesgc/chembiohub_helpdesk#987]
I couldn't find it, I'm pretty sure no one else will either. We need to make this clearer for the user to identify.

**Comment by KarenPorter** [comment]
agree.  providing access to it from the handsontable tool bar would make this more obvious (between Edit and Add single options would make sense to me).
